### PR TITLE
Add a simple python hello world example.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: examples/python2.7/BUILD [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    srcs = ["hello.py"],
+    main = "hello.py",
+)
+
+docker_build(
+    name = "hello",
+    base = "//python2.7:python27",
+    cmd = [
+        "/hello_py.par",
+        "/etc",
```
Comment (mattmoor) [inline examples/python2.7/BUILD]: Can you document why you are passing `/etc` here?  It is weird and an abundance of documentation wouldn't hurt.

File: examples/python2.7/hello.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/python
```
Comment (mattmoor) [inline examples/python2.7/hello.py]: License header

File: examples/python2.7/BUILD [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    ],
+    files = [":hello_py.par"],
+)
```
Comment (mattmoor) [inline examples/python2.7/BUILD]: Do you want to add a `structure_test` that executes the sample with expected output?

Comment (dlorenc) [inline examples/python2.7/BUILD]: We already test everything important in the python base structure test, I'd rather leave this one the way it is.

File: examples/python2.7/hello.py (hunk 2 above)
Comment (dlorenc) [inline examples/python2.7/hello.py]: Done.

File: examples/python2.7/BUILD (hunk 1 above)
Comment (dlorenc) [inline examples/python2.7/BUILD]: Done. Basically ls'ing / was too much output :)

**Pull request merged**